scoring.js

```diff
@@ -48,6 +48,6 @@ module.exports.calculateScore = function (player) {
     case 'TE':
       return calculateReceivingScore(player)
     default:
-      return 0
+      throw new Error('Unknown player position (K): ${product.type}')
```

Comment:
In the case of this player the error would read. `Unknown player position (K): K` which is a bit odd. If we passed in a player with position LB we would get `Unknown player position (K): LB` which would be very confusing.